pom.xml

```diff
@@ -27,6 +27,9 @@
                     <source>11</source>
                     <target>11</target>
                     <release>11</release>
+                    <compilerArguments>
```

Comment:
wat

Reply:
In order for GraphQL to recognise method parameter names, we need to add this

Reply:
again wat. I understand why this is here now but seriously graphql

Reply:
It's because normally method parameters don't have reflection metadata so it turns out as `arg0`, `arg1` etc. We could do it without method parameters by using more annotations but I think this is easier and keeps the code cleaner. So it's Java's fault in this case